recipe/meta.yaml

```diff
@@ -25,14 +25,10 @@ requirements:
 
 test:
   commands:
-    - set vex
-    - which nim
+    - command -v nim
```

Comment:
Was `set -vex` redundant? Or simply misspelled? (Missing `-`?)

Reply:
Yes both. The tests are run with `-ex` and don't receive any input, i.e., `-v` is not needed.

Reply:
Thinking about it, the `-v` part I wrote isn't correct, I think. But `-v` should be redundant with `-x` anyway.

Reply:
I think `-v` shows the shell script before substitutions and `-x` after. I guess `-v` is not essential here.

Reply:
Yes, agreed to both points ;).